modules/programs/neovim.nix

```diff
@@ -241,9 +274,12 @@ in {
 
     home.packages = [ cfg.finalPackage ];
 
-    xdg.configFile = mkIf (neovimConfig.neovimRcContent != "") {
-      "nvim/init.vim".text = neovimConfig.neovimRcContent;
+    xdg.configFile."nvim/init.vim" = mkIf (neovimConfig.neovimRcContent != "") {
+      text = neovimConfig.neovimRcContent;
```

Comment:
Does this end up creating the `init.vim` file even if the contents were supposed to be empty?

Reply:
I have just tested it and it won't create `init.vim` when condition is `false`. However it actually does create file due to #1907